src/UNL/UCBCN/Manager/Controller.php

```diff
@@ -205,6 +206,26 @@ public function getCalendars() {
         return $user->getCalendars();
     }
 
+    public function hasPermission($permission_description, $calendar_id) 
+    {
+        switch ($permission_description) {
```

Comment:
Namespaces complicate it a little bit. Best to use the fully-qualified name `\UNL\UCBCN\Permission::`